codegen/types_test.go

```diff
@@ -26,18 +26,23 @@ func TestGoTypeDef(t *testing.T) {
 				{"IntField", &expr.AttributeExpr{Type: expr.Int, DefaultValue: 1}},
 				{"StringField", &expr.AttributeExpr{Type: expr.String, DefaultValue: "foo"}},
 			}}
-		ut         = &expr.UserTypeExpr{AttributeExpr: &expr.AttributeExpr{Type: expr.Boolean}, TypeName: "UserType"}
-		rt         = &expr.ResultTypeExpr{UserTypeExpr: &expr.UserTypeExpr{AttributeExpr: &expr.AttributeExpr{Type: expr.Boolean}, TypeName: "ResultType"}, Identifier: "application/vnd.goa.example", Views: nil}
-		userType   = &expr.AttributeExpr{Type: ut}
-		resultType = &expr.AttributeExpr{Type: rt}
-		mixedObj   = &expr.AttributeExpr{
+		ut                     = &expr.UserTypeExpr{AttributeExpr: &expr.AttributeExpr{Type: expr.Boolean}, TypeName: "UserType"}
+		rt                     = &expr.ResultTypeExpr{UserTypeExpr: &expr.UserTypeExpr{AttributeExpr: &expr.AttributeExpr{Type: expr.Boolean}, TypeName: "ResultType"}, Identifier: "application/vnd.goa.example", Views: nil}
+		userType               = &expr.AttributeExpr{Type: ut}
+		resultType             = &expr.AttributeExpr{Type: rt}
+		stringMetaType         = expr.MetaExpr{"struct:field:type": []string{"string"}}
+		jsonWithImportMetaType = expr.MetaExpr{"struct:field:type": []string{"json.RawMessage", "encoding/json"}}
+		jsonWithRenameMetaType = expr.MetaExpr{"struct:field:type": []string{"jason.RawMessage", "encoding/json", "jason"}}
```

Comment:
`jason` typo? Or is this intentional?

Reply:
Ah I see. This shows overriding package name?

Reply:
Yes, this is jst testing the 3rd parameter, qualified package